openprescribing/dmd2/templates/dmd/vmp_relationships.html

```diff
@@ -55,18 +84,19 @@ <h2>VMPPs, AMPs, AMPPs</h2>
   </tr>
   {% else %}
   <tr>
+    {% if forloop.counter0 == 1 %}<th class="amp-header"   rowspan="{{ amp_rows }}"><div class="vertical">AMP/brand/supplier</div></th> {% endif %}
```

Comment:
"AMP/brand/supplier" Again, I think I'd rather this just said "AMP", and describe what this means in the documentation.